### ISSUE TITLE
class << self syntax not working for methods

### ISSUE THREAD

**Issue by deanobarnett** [mbj/mutant#258]
When writing a class method like this

``` ruby
class Foo
  class << self
    def bar
      p 'hello!'
    end
  end
end
```

Mutant throws an error saying that it cannot find method definition of Foo.bar

**Comment by mbj** [comment]
I like the idea, but I do not have a need to do it for my projects. So tagging as not-on-authors-free-time to encourage funding or (preferred) outside contribution.

**Comment by mbj** [comment]
This was ultimatively addressed in #1009, as a version of #1000. Big thanks for @telyn for the hard work.

Released in mutant-0.9.9.
